js/currencycom.js

```diff
@@ -142,12 +141,22 @@ module.exports = class currencycom extends Exchange {
                         'v1/klines',
                         'v1/ticker/24hr',
                         'v2/time',
-                        'v2/exchangeInfo', // everything similar as v1, except serverTime
+                        'v2/exchangeInfo',
                         'v2/depth',
                         'v2/aggTrades',
                         'v2/klines',
                         'v2/ticker/24hr',
-                        //
+                    ],
+                },
+                'private': {
+                    'get': [
+                        'v1/currencies',
+                        'v2/currencies',
+                        'v1/leverageSettings',
+                        'v1/openOrders',
+                        'v1/tradingPositions',
+                        'v1/account',
+                        'v1/myTrades',
```

Comment:
oh, how I missed this. probably when cutting 'currencies' i've forgot to move other private endpoints from public. silly mistake, sorry.